doc/_data/schemas/common/PostureTask_targets_properties.json

```diff
@@ -20,6 +20,15 @@
         "$ref": "/../../Tasks/JointGains.json" 
       } 
     },
+    "jointWeights":
+    {
+      "type": "object",
+      "description": "Weights for specific joints",
+      "properties":
+      {
+        "*": { "type": "number", "description": "Joint weight [rad]" }
```

Comment:
Not `[rad]`

Reply:
Thanks, fixed in [`c40ba95` (#316)](https://github.com/jrl-umi3218/mc_rtc/pull/316/commits/c40ba9599a06a89c5884c2f634d606d5fe1d752d)